templates/home/_fsf_go.html

```diff
@@ -30,7 +30,9 @@ <h4>Here’s how httplab uses it:</h4>
     <b>go-importpath</b>: github.com/gchaincl/httplab
     <b>source</b>: .
     <b>source-type</b>: git
-
+  <b> build-packages</b>:
```

Comment:
Indentation is off:
<img width="494" alt="screenshot 2019-02-11 at 15 13 57" src="https://user-images.githubusercontent.com/1523179/52572443-bbb37c00-2e0f-11e9-8f98-bd0789a384ea.png">